tests/Microsoft.Bot.Expressions.Tests/BadExpressionTests.cs

```diff
@@ -64,7 +64,6 @@ public class BadExpressionTests
             Test("split(one, 'l')"), // split only accept string parameter
             Test("split(hello, 1)"), // split only accept string parameter
             Test("substring(hello, 0.5)"), // the second parameter of substring must be integer
-            Test("substring(one, 0)"), // the first parameter of substring must be string
             Test("substring(hello, 10)"), // the start index is out of the range of the string length
```

Comment:
Why did you remove?

Reply:
because one is an 1.0.  substring() does a tryevaluate<string>() which will give you "1.0" as a string, so this unit test does not fail anymore.

---
In reply to: [367284789](https://github.com/microsoft/botbuilder-dotnet/pull/3253#discussion_r367284789) [](ancestors = 367284789)